FoxHound.Client/src/AddBlog.tsx

```diff
@@ -0,0 +1,108 @@
+import React, { useState, useEffect, ChangeEvent, FormEvent } from "react";
+import axios from "axios";
+import { makeStyles } from "@material-ui/core/styles";
+import TextField from "@material-ui/core/TextField";
+import Button from "@material-ui/core/Button";
+import Grid from "@material-ui/core/Grid";
+import Paper from "@material-ui/core/Paper";
+import Container from "@material-ui/core/Container";
+import Box from "@material-ui/core/Box";
+import Typography from "@material-ui/core/Typography";
+
+interface IProps {
+  refreshBlogList: () => void;
+}
+
+const useStyles = makeStyles((theme) => ({
+  root: {
+    padding: theme.spacing(2, 3),
+  },
+  templateBuilder: {
+    marginTop: theme.spacing(2),
+    marginBottom: theme.spacing(2),
+  },
```

Comment:
Remove this as it looks like it isn't being used anywhere.

Reply:
Removed, thanks